.gitignore

```diff
@@ -154,5 +154,9 @@ sourcemap.js
 .jest/
 coverage/
 
+# Firebase config
+GoogleService-Info.plist
+google-services.json
```

Comment:
Leaving the credentials of out git. These files will be written during CI, and we'll each have to download the dev versions for local development.